### PR TITLE
Move translation sources to CONTRIBUTING

### LINKED ISSUE #210: Clarify overall translations guidelines
[TRANSLATION_SOURCES](https://github.com/axelboc/anki-ultimate-geography/blob/master/TRANSLATION_SOURCES.md) should clarify the overall guideline of following the main Wikipedia site for each language when their contents differ (notably the article title and the capital(s) listed in the infobox).

We should also mention that:

- This guideline doesn't apply to flags, since translating the _Flag_ field seems a lot of effort for just a few minor differences (notable state vs. civil flags)...
- If there hasn't been discussions about the differences on Wikipedia, then it's always good to start one.

### PR DESCRIPTION
**Pull request by axelboc**
Fix #210 

Since the content guidelines explain (for each field) that Wikipedia is our default source, I don't think it's worth repeating that in `TRANSLATION_SOURCES.md`. I think it makes more sense to move the sources into `CONTRIBTUTING.md`, right after the content guidelines. What do you reckon?

I've also updated the table of contents and made some minor stylistic tweaks to the Norwegian Bokmal section following #279 (present tense, translated source titles only, etc.)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -171,11 +181,22 @@ The following guidelines apply to map images:
 - The style (colours, strokes, etc.) should match that of existing maps in the deck (note that water bodies use a different style than countries).
 - For small islands or archipelagos, the map should include a zoom box to facilitate identification.
 
-### Translation sources
+## Translation sources
+
+When Wikipedia in a given language is not sufficently exhaustive to support the translation of this deck, additional sources may be used. This sections references such sources, as well as any stylistic choices made by translators.
```
Comment (aplaice) [inline CONTRIBUTING.md:186]: s/sections/section/

```suggestion
When Wikipedia in a given language is not sufficently exhaustive to support the translation of this deck, additional sources may be used. This section references such sources, as well as any stylistic choices made by translators.
```

**Comment by aplaice** [comment]
I think it (and the tweaking of the Norwegian section) make sense!

<hr/>

Regarding the TOC, it might perhaps make sense to indent everything by one, since now with the additional sub-sections it looks slightly weird ([current](https://github.com/axelboc/anki-ultimate-geography/blob/2edc6dd46edd13f528b27a598eda293eb3985386/CONTRIBUTING.md#table-of-contents), [updated](https://github.com/axelboc/anki-ultimate-geography/blob/535994e5bb448ce31760c77840d1dec5a34903e8/CONTRIBUTING.md#table-of-contents)).

(I'm not entirely sure though.)

```md
- [Contributor's guide](#contributors-guide)
    - [Set-up](#set-up)
    - [Build and import](#build-and-import)
    - [Indexing](#indexing)
    - [Quotes normalisation](#quotes-normalisation)

- [Content guidelines](#content-guidelines)
    - [_Country_ field](#country-field)
    - [_Country info_ field](#country-info-field)
    - [_Capital_ field](#capital-field)
    - [_Capital info_ field](#capital-info-field)
    - [_Capital hint_ field](#capital-hint-field)
    - [_Flag_ field](#flag-field)
    - [_Flag similarity_ field](#flag-similarity-field)
    - [_Map_ field](#map-field)

- [Translation sources](#translation-sources)
    - [Norwegian Bokmål](norwegian-bokmål)

- [Maintainer's guide](#maintainers-guide)
    - [Versioning](#versioning)
    - [Release process](#release-process)
```

<hr/>

- [Contributor's guide](#contributors-guide)
    - [Set-up](#set-up)
    - [Build and import](#build-and-import)
    - [Indexing](#indexing)
    - [Quotes normalisation](#quotes-normalisation)

- [Content guidelines](#content-guidelines)
    - [_Country_ field](#country-field)
    - [_Country info_ field](#country-info-field)
    - [_Capital_ field](#capital-field)
    - [_Capital info_ field](#capital-info-field)
    - [_Capital hint_ field](#capital-hint-field)
    - [_Flag_ field](#flag-field)
    - [_Flag similarity_ field](#flag-similarity-field)
    - [_Map_ field](#map-field)

- [Translation sources](#translation-sources)
    - [Norwegian Bokmål](norwegian-bokmål)

- [Maintainer's guide](#maintainers-guide)
    - [Versioning](#versioning)
    - [Release process](#release-process)

**Comment by axelboc** [comment]
How about listing only the top headings?

**Comment by aplaice** [comment]
> How about listing only the top headings?

That might be a shame since the full TOC is useful as a form of summary of the readme, for somebody who's just scanning it.

Perhaps let's just ignore my comment and leave the updated TOC as it is! :D 

Happy Easter / Pesach / Holidays!

**Comment by axelboc** [comment]
Alright, no worries. I've bolded the first-level links for added clarity.

**Pull request merged**